cmd/collector/main.go

```diff
@@ -96,15 +96,14 @@ func main() {
 				HealthCheck:    svc.HC(),
 			})
 			collectorOpts := new(app.CollectorOptions).InitFromViper(v)
-			c.Start(collectorOpts)
+			_ = c.Start(collectorOpts)
```

Comment:
if you'd like to clean this up even more, c.Start currently never returns an error, because it logs Fatals. Instead, it should be returning wrapped errors and log.Fatal should only happen here.